package.json

```diff
@@ -21,6 +21,7 @@
     "@types/jest-expect-message": "^1.0.3",
     "@typescript-eslint/eslint-plugin": "^5.28.0",
     "@typescript-eslint/parser": "^5.28.0",
+    "chalk": "^4.1.2",
```

Comment:
Minor: We don't need `chalk` as a dev-dependency in the project root.

Reply:
removed the dependency from project root